src/tools/gcc.lua

```diff
@@ -284,12 +284,20 @@
 -- Decorate include file search paths for the GCC command line.
 --
 
-	function gcc.getincludedirs(cfg, dirs, sysdirs)
+	function gcc.getincludedirs(cfg, dirs, sysdirs, frameworkdirs)
 		local result = {}
-		for _, dir in ipairs(dirs) do
+		for _, dir in ipairs(dirs or {}) do
 			dir = project.getrelative(cfg.project, dir)
 			table.insert(result, '-I' .. p.quoted(dir))
 		end
+
+		if (table.contains(os.getSystemTags(cfg.system), "darwin")
+			and cfg.frameworkdirs ~= nil and #cfg.frameworkdirs) then
+			for _, dir in ipairs(frameworkdirs or {}) do
+				table.insert(result, '-F' .. p.quoted(dir))
```

Comment:
This isn't consistent with the other blocks, the others use `project.getrelative()` on `dir`, why doesn't this one?

Reply:
During tests I always used absolute path so i didn't catch my attention. I will made more tests to ensure relative paths are accepted by the -F option